include/swift/Driver/ToolChain.h

```diff
@@ -29,6 +29,9 @@ namespace driver {
   class Job;
   class OutputInfo;
 
+/// The limit for passing a list of files on the command line.
+const size_t TOO_MANY_FILES = 128;
```

Comment:
Please make this `static`, there's no need for it to have a stable address.

Reply:
Ok